src/components/FeeSelector.vue

```diff
@@ -3,7 +3,10 @@
     <label class="btn btn-option btn-option-lg"
         v-for="name in ['slow', 'average', 'fast']" :key="name"
         :class="{ active: (name === value)}"
-        v-tooltip="{ content: getTooltip(name)} "
+        v-tooltip="{
+          content: getTooltip(name),
+          trigger: txTypes ? 'hover' : 'manual'
```

Comment:
this would mean there is no tooltip displayed I guess? 

in stead can we do this? 

at `if (this.txTypes) {`
We can do `else` that displays `this.fees[name].fee`.
for example `55 gwei`

Reply:
fixed